packages/strapi-helper-plugin/lib/src/components/InputNumber/index.js

```diff
@@ -29,7 +29,7 @@ function InputNumber(props) {
           onFocus={props.onFocus}
           placeholder={message}
           ref={props.inputRef}
-          step={!Number.isNaN(Number(props.step)) ? Number(props.step) : 1}
+          step={!Number.isNaN(Number(props.step)) ? Number(props.step) : 'any'}
```

Comment:
Apparently this step line make the component buggy on chrome.

`step={props.step}` seems to make the component works in Chrome, Firefox and Safari

Reply:
Cleans it up too. I've updated this line.